### PR TITLE
Add 'Ethereum Ecosystem' CTA to ethereum.org

### LINKED ISSUE #12372: Add a Call To Action to Ethereum Ecosystem
### Is your feature request related to a problem? Please describe.

Currently, visitors to https://ethereum.org/en/dapps/ may not be aware of the expansive list of dApps available on Ethereum and its Layer 2s. I find it limiting that users aren't guided to resource(s) that showcase the full range of available applications.

### Describe the solution you'd like

I propose adding a new CTA that directs users to the '[Ethereum Ecosystem](https://www.ethereum-ecosystem.com/)' page. This page features over 1000 dApps across Ethereum and its Layer 2 networks, such as Optimism, Base, and Starknet.

### Describe alternatives you've considered

N/A

### Additional context

'Ethereum Ecosystem' has received financial support from the Optimism collective during grant season 4 and RetroPGF3, emphasizing its significance and community trust. It's also being listed by Starknet [here](https://www.starknet.io/en/ecosystem/dapps). 

By featuring it more prominently, we can ensure it gets the attention it deserves and can be utilized by more users.

### Would you like to work on this issue?

- [X] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by Maxservais**
This PR introduces a Call to Action on https://ethereum.org/en/dapps/, guiding visitors to [Ethereum Ecosystem](https://www.ethereum-ecosystem.com), an unofficial ecosystem page for Ethereum and its L2s.

## Description
This addition aims to provide enhanced visibility for a resource that catalogues over 1000 dApps across Ethereum and its Layer 2s, facilitating greater user exploration and engagement within the ecosystem.

Capture d’écran 2024-03-05 à 15 25 03

## Related Issue
- Closes #12372

## Preview URL
- https://deploy-preview-12373--ethereumorg.netlify.app/en/dapps

## Summary by CodeRabbit

- **New Features**
	- Introduced a new section in the Dapps page to explore the ecosystem, featuring a callout banner with a direct link for user engagement.

### CODE REVIEW

**Comment by konopkja** [comment]
This is too aggressive ad, it is above the list of curated dapps on the page itself. It is a worthy addition for users who seek an expanded list of dapps or to see the whole ecosystem, but will not be useful to a beginner who needs less options nad more curation, which https://www.ethereum-ecosystem.com/ currently does not offer.

I suggest rethinking how we link to this project on this page.

**Comment by Maxservais** [comment]
Hi @konopkja ,

Thanks for the constructive feedback on the CTA.

I see your point about its prominence possibly being overwhelming for newcomers. Inspired by the L2Beat CTA on ethereum.org's [Layer 2 overview](https://ethereum.org/en/layer-2/), I was aiming for similar visibility for Ethereum Ecosystem.
image


Would you be open to placing the CTA lower on the page, where it might be more contextually appropriate for users looking for expanded information? I'm keen to adjust the approach to better fit the community's needs.

Looking forward to your guidance.

**Comment by Maxservais** [comment]
@konopkja what do you think about this?

image

**Comment by konopkja** [comment]
> @konopkja what do you think about this?
> 
> image

its better, though this is a very big recognition - using a full width banner like this, there are only few instances when we have done that before and those were open source communities not a closed project so I am still hesitant on this approach. For now lets keep this open and see what others say.

**Comment by wackerow** [comment]
image

I do like the idea of linking out to this, but am hesitant on using such a large callout for an initial listing. To me this feels like a good follow-up to the listings shown above, like "Want to browse more?"

image

^ Referring to this space here, as a regular text callout.

Also, ideally we make it clear to users that they're going to an external resource. "Ethereum.org" "Ethereum" "Ethereum Foundation" "Ethereum Ecosystem"... User's often don't discern the differences well.

**Comment by konopkja** [comment]
agree with @wackerow and i would suggest changing it to a text based link perhaps

**Comment by Maxservais** [comment]
Hello @wackerow @konopkja,

Thank you both for your valuable input. Based on your suggestions, I’ve developed a more subdued approach: a button that prompts users who are interested in delving deeper after browsing the listings. This replaces the more assertive banner with a subtler call to action.

Additionally, a 'tooltip' will appear when users hover over the button, alerting them that they are about to visit a third-party site not managed by Ethereum.org.

Here's a preview of the implementation:
image

I’m eager to hear your thoughts on this update.

File: src/pages/dapps.tsx [hunk 1]
```diff
@@ -1550,6 +1551,14 @@ const DappsPage = () => {
                 />
               </RightColumn>
             </TwoColumnContent>
+            <Box>
+              <ButtonLink
+                to="https://www.ethereum-ecosystem.com/apps"
+                title={t("page-dapps:page-dapps-explore-1")}
+              >
+                {t("page-dapps:page-dapps-explore-title")}
+              </ButtonLink>
```
Comment (wackerow) [inline src/pages/dapps.tsx:1560]: @konopkja and I were both suggesting a text link here... but will ultimately defer to him on this.

As for the usage of `title`, I'd say this is a very subtle way to let people know, while I was imagining something much more obvious and attention grabbing where that notice cannot be missed. @konopkja ideas?

Maybe:

```
Want to browse more? [Check out thousands of dapps on the Ethereum Ecosystem\* website](https://www.ethereum-ecosystem.com/apps)

_(\*Ethereum Ecosystem is not affiliated with ethereum.org or the Ethereum Foundation)_
```

Comment (Maxservais) [reply src/pages/dapps.tsx]: Hello @wackerow,
Sorry I misunderstood the text link suggestion earlier.

I've spent about an hour attempting to configure it in a way that ensures the warning stands out next to the text link, but I must admit it's proving quite challenging to balance the aesthetics with the practicality of the warning notice.

@konopkja @wackerow What do you think of this?

image

Thank you for your understanding and patience so far as we strive to create a solution that is user-friendly and also clearly communicates that Ethereum Ecosystem is independent of Ethereum.org.

Comment (wackerow) [reply src/pages/dapps.tsx]: ping @konopkja... I would personally keep the link on the same line as the text; we use the pattern a lot and typically don't line-break. The warning, however, I think looks good.

Comment (konopkja) [reply src/pages/dapps.tsx]: 1. yeah i think this needs some styling to do to look appropriate on the page
1A.  "Want to browse more?" should be bold and "want to browse more apps?", 1B. the link phrase is too long - remove "on Ethereum Ecosystem's website"
3. the disclaimer isnt necessary, we dont provide it for other external links either

Comment (Maxservais) [reply src/pages/dapps.tsx]: Thank you both @konopkja @wackerow for the feedbacks! I have made the necessary changes

**Review by corwintines — COMMENTED** [review]
Thanks @Maxservais! 

In general I think this is ready to go. Going to bring this up next GitHub grooming session to see if there are any blockers/reservations left with what is here, and if not get this wrapped up.

File: src/intl/en/page-dapps.json [hunk 2]
```diff
@@ -285,5 +285,7 @@
   "page-dapps-dapp-description-dodo": "DODO is a on-chain liquidity provider, which leverages the Proactive Market Maker algorithm (PMM)",
   "page-dapps-dodo-image-alt": "DODO logo",
   "page-dapps-dapp-description-artblocks": "Art Blocks is dedicated to bringing compelling works of contemporary generative art to life",
-  "page-dapps-artblocks-image-alt": "Art Blocks logo"
+  "page-dapps-artblocks-image-alt": "Art Blocks logo",
+  "page-dapps-explore-title": "Want to browse more apps?",
+  "page-dapps-explore": "Check out hundreds of dapps"
```
[the remark this answers is not in the document]
Comment (wackerow) [reply src/intl/en/page-dapps.json:290]: Fine either way here... "hundreds" does kinda drive home the point that there are a lot of them

**Review by corwintines — APPROVED** [review]

**Pull request merged**